src/History/CommitteeMembershipHistoryHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace AppBundle\History;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Repository\CommitteeMembershipHistoryRepository;
+use Cake\Chronos\Chronos;
+
+class CommitteeMembershipHistoryHandler
+{
+    private $historyRepository;
+
+    public function __construct(CommitteeMembershipHistoryRepository $historyRepository)
+    {
+        $this->historyRepository = $historyRepository;
+    }
+
+    public function queryCountByMonth(Adherent $referent, int $months = 6): array
+    {
+        foreach (range(0, $months - 1) as $monthInterval) {
```

Comment:
as an alternative:
```
$startDate = (new Chronos('last day of -6 month'))->setTime(23, 59, 59, 999);
$endDate = new Chronos();

$period = new \DatePeriod($startDate, new \DateInterval('P1M'), $endDate);

foreach ($period as $date) {
    // $date -> each month
}
```

Reply:
I like that !

Reply:
Sadly it does not work as expected :-(

![image](https://user-images.githubusercontent.com/895123/40720993-9107039e-6418-11e8-94c7-ac8c5353f6c9.png)

![image](https://user-images.githubusercontent.com/895123/40721029-a0c1cc06-6418-11e8-97af-c665092f54c1.png)

Reply:
easier to read https://3v4l.org/C0JbP

Reply:
😢 👍